packages/general/src/time/Time.ts

```diff
@@ -29,20 +30,46 @@ export class Time {
     }
     static readonly nowMs = (): number => Time.get().nowMs();
 
-    /** Returns a timer that will call callback after durationMs has passed. */
+    /**
+     * Create a timer that will call callback after durationMs has passed.
+     */
     getTimer(name: string, durationMs: number, callback: Timer.Callback): Timer {
         return new StandardTimer(name, durationMs, callback, false);
     }
     static readonly getTimer = (name: string, durationMs: number, callback: Timer.Callback): Timer =>
         Time.get().getTimer(name, durationMs, callback);
 
-    /** Returns a timer that will periodically call callback at intervalMs intervals. */
+    /**
+     * Create a timer that will periodically call callback at intervalMs intervals.
+     */
     getPeriodicTimer(name: string, intervalMs: number, callback: Timer.Callback): Timer {
         return new StandardTimer(name, intervalMs, callback, true);
     }
     static readonly getPeriodicTimer = (name: string, intervalMs: number, callback: Timer.Callback): Timer =>
         Time.get().getPeriodicTimer(name, intervalMs, callback);
 
+    /**
+     * Create a promise that resolves after a specific interval or when canceled, whichever comes first.
+     */
+    wait(name: string, durationMs: number): CancelablePromise {
```

Comment:
Please lets adjust sleep and not add wait :-)

Reply:
done